matrix/matrices_test.go

```diff
@@ -161,14 +162,14 @@ func TestReqGraph_createMatrix(t *testing.T) {
 		},
 		Document: &sddDoc,
 	}
-	rg.Reqs["REQ-TEST-SWL-3"] = &Req{
+	rg.Reqs["REQ-TEST-SWL-3"] = &reqs.Req{
 		ID:        "REQ-TEST-SWL-3",
 		IDNumber:  3,
 		ParentIds: []string{},
 		Document:  &sddDoc,
 	}
 
-	errs := rg.resolve()
+	errs := rg.Resolve()
```

Comment:
I'm not sure I lean on making it private in the future, because for the LSP functionality you'd want to be able to resolve diagnostics for single files (be it markdown or code), but probably not through this specific API. I added the comment for now.